test/api/helper.cpp

```diff
@@ -23,16 +23,32 @@ namespace WasmEdge {
 
 namespace {
 Proposal ProposalList[] = {
-    Proposal::TailCall, Proposal::MultiMemories,      Proposal::Annotations,
-    Proposal::Memory64, Proposal::ExceptionHandling,  Proposal::ExtendedConst,
-    Proposal::Threads,  Proposal::FunctionReferences, Proposal::GC,
-    Proposal::RelaxSIMD};
+    // Should be in reversed order due to the proposal dependencies.
```

Comment:
Maybe we can use topological sort here?

Reply:
It'll be too complicated.
The turned-on and turned-off proposals are not related in every cases.